ui/.eslintrc.json

```diff
@@ -1,16 +0,0 @@
-{
-    "parserOptions": {
-        "ecmaVersion": 6,
-        "sourceType": "module",
-        "ecmaFeatures": {
-            "jsx": true
-        }
-    },
-    "plugins": [
-        "codice-linter"
-    ],
-    "rules": {
-        "codice-linter/no-absolute-urls": 2
```

Comment:
Where does this live now?

Reply:
https://github.com/connexta/eslint-plugin